packages/sui-studio/test/server/integrationSpec.js

```diff
@@ -0,0 +1,193 @@
+const fs = require('fs')
+const path = require('path')
+const util = require('util')
+const {expect} = require('chai')
+const childProcess = require('child_process')
+const exec = util.promisify(childProcess.exec)
+
+const SUI_STUDIO_BINARY_DIR = path.join(__dirname, '..', '..', 'bin')
+const SAMPLE_STUDIO_PATH = path.join(__dirname, 'integration', 'sample-studio')
+const EMPTY_STUDIO_PATH = path.join(__dirname, 'integration', 'empty-studio')
+
+const safeRemoveDir = uri => {
+  // console.log('Removing', uri)
+  try {
+    fs.rmdirSync(uri, {recursive: true})
+  } catch (e) {
+    // console.error(e.message)
+  }
+}
+
+describe('[Integration] sui-studio', () => {
+  it('Should compile and build a static version with one component', async function() {
+    this.timeout(0)
+
+    const {stdout: stdoutStudioBuild} = await exec(
+      `node "${SUI_STUDIO_BINARY_DIR}/sui-studio-build" -C`,
+      {
+        cwd: SAMPLE_STUDIO_PATH,
+        env: {
+          ...process.env,
+          PWD: SAMPLE_STUDIO_PATH
+        }
+      }
+    )
+
+    expect(stdoutStudioBuild.includes('Error')).to.be.false
+  })
+
+  it('Should start a studio in mode dev', function(done) {
+    this.timeout(0)
+    let server
+    try {
+      server = childProcess.spawn(
+        'node',
+        [`${SUI_STUDIO_BINARY_DIR}/sui-studio-dev`, 'atom/button'],
+        {
+          detached: false,
+          env: {
+            ...process.env,
+            PWD: SAMPLE_STUDIO_PATH
+          }
+        }
+      )
+      // server.stdout.pipe(process.stdout)
+      // server.stderr.pipe(process.stdout)
```

Comment:
👀

Reply:
I wanted to leave these comments as "documentation" to know how to get this information printed by console